localstack/services/lambda_/provider.py

```diff
@@ -1136,6 +1135,21 @@ def update_function_code(
 
         old_function_version = function.versions.get("$LATEST")
         replace_kwargs = {"code": code} if code else {"image": image}
+
+        if architectures := request.get("Architectures"):
+            if len(architectures) != 1:
+                raise ValidationException(
+                    f"1 validation error detected: Value '[{', '.join(architectures)}]' at 'architectures' failed to "
+                    f"satisfy constraint: Member must have length less than or equal to 1",
+                )
```

Comment:
```suggestion
            if len(architectures) > 1:
                raise ValidationException(
                    f"1 validation error detected: Value '[{', '.join(architectures)}]' at 'architectures' failed to "
                    f"satisfy constraint: Member must have length less than or equal to 1",
                )
```

Shouldn't be an empty architectures list be fine according to the error message? (would also affect the `architectores[0]` below since then it won't be safe anymore.

Reply:
If that's not the case (bc AWS treats it as null and thus returns the error msg below), please add a comment :+1:

Reply:
Added a comment:

```py
# An empty list of architectures is also forbidden. Further exceptions are tested here for create_function:
# tests.aws.services.lambda_.test_lambda_api.TestLambdaFunction.test_create_lambda_exceptions
```